druid/src/contexts.rs

```diff
@@ -351,18 +351,23 @@ impl_context_method!(EventCtx<'_, '_>, UpdateCtx<'_, '_>, LifeCycleCtx<'_, '_>,
         self.state.set_menu(menu);
     }
 
-    /// Create a new sub window that will have its app data synchronised with the nearest surrounding widget pod.
-    /// 'U' must be the type of the nearest surrounding widget pod. The 'data' argument should be the current value of data
-    /// for that widget.
+    /// Create a new sub-window.
+    ///
+    /// The sub-window will have its app data synchronised with caller's nearest ancestor [`WidgetPod`].
+    /// 'U' must be the type of the nearest surrounding [`WidgetPod`]. The 'data' argument should be
+    /// the current value of data  for that widget.
+    ///
+    /// [`WidgetPod`]: struct.WidgetPod.html
     // TODO - dynamically check that the type of the pod we are registering this on is the same as the type of the
-    // requirement. Needs type ids recorded. This goes wrong if you don't have a pod between you and a lens.
+    // requirement. Needs type ids recorded. This goes wrong if you don't have a pod between you and a lens.    pub fn new_sub_window<W: Widget<U> + 'static, U: Data>(
```

Comment:
is the diff screwed up or is there a trailing bit of code at the end of this line?

Reply:
I think a weird C&P error